src/Home/Home.tsx

```diff
@@ -66,25 +68,29 @@ const EntriesView = () => {
         </tr>);
     }
 
-    return <table>
-        <thead>
-            <tr>
-                <th>Date</th>
-                <th>Title</th>
-                <th>Hours Active</th>
-                <th>Hours Sleeping</th>
-                <th>Hours Focused</th>
-                <th>Hours on Screen</th>
-                <th>Hours Outside</th>
-                <th>Hours Reading</th>
-                <th>Mood</th>
-                <th>Contents</th>
-                <th>Created</th>
-                <th>Updated</th>
-            </tr>
-        </thead>
-        <tbody>{tr_rows}</tbody>
-    </table>;
+    return (
+        <div>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Date</th>
+                        <th>Title</th>
+                        <th>Hours Active</th>
+                        <th>Hours Sleeping</th>
+                        <th>Hours Focused</th>
+                        <th>Hours on Screen</th>
+                        <th>Hours Outside</th>
+                        <th>Hours Reading</th>
+                        <th>Mood</th>
+                        <th>Contents</th>
+                        <th>Created</th>
+                        <th>Updated</th>
+                    </tr>
+                </thead>
+                <tbody>{tr_rows}</tbody>
+            </table>
+        </div>
+    );
```

Comment:
why the additional `div`?

Reply:
![div](https://github.com/user-attachments/assets/e6e0c51c-b157-4526-bae6-d940d486983f)
that for the title under the Calendar

Reply:
i added the original one 
    return <table>
        <thead>
            <tr>
                <th>Date</th>
                <th>Title</th>
                <th>Hours Active</th>
                <th>Hours Sleeping</th>
                <th>Hours Focused</th>
                <th>Hours on Screen</th>
                <th>Hours Outside</th>
                <th>Hours Reading</th>
                <th>Mood</th>
                <th>Contents</th>
                <th>Created</th>
                <th>Updated</th>
            </tr>
        </thead>
        <tbody>{tr_rows}</tbody>
    </table>;
};